app/models/manageiq/providers/monitoring_manager.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers
+  class MonitoringManager < BaseManager
+    class << model_name
+      def route_key
+        "ems_monitoring"
```

Comment:
Actually a routing isn't needed here as this provider should not have separate screens